docs/styling.md

```diff
@@ -104,6 +104,12 @@ If you write a new theme by CSS styling please contribute it back to FSharp.Form
 You can do advanced styling by creating a new template.  Add a file `docs/_template.html`, likely starting
 with the existing default template.
 
+to enable hot reload during development with `fsdocs watch` in a custom `_template.html` file, make sure to add the following line to your `<head>` tag:
```

Comment:
Capital "T" at start of line.

Also there is a table of substitutions in content.md I believe, this should be added there